src/clojure_tic_tac_toe/core.clj

```diff
@@ -1,16 +1,31 @@
 (ns clojure-tic-tac-toe.core
-  (:gen-class))
+  (:require [clojure-tic-tac-toe.input_output :as io]
+            [clojure-tic-tac-toe.board :as board]))
 
-(defn board-to-string []
-  (clojure.string/join "\n" [" 1 | 2 | 3 "
-                             "---+---+---"
-                             " 4 | 5 | 6 "
-                             "---+---+---"
-                             " 7 | 8 | 9 "]))
+(defn get-move
+  [player]
+  (io/prompt-player-for-move player)
+  (flush)
+  (io/get-move))
+
+(defn int-to-keyword
+  [integer]
+  (keyword (str integer)))
+
+(defn play-round
+  [board player]
+  (def move (get-move player))
+  (def move-as-keyword (int-to-keyword move))
+  (def board-with-move (board/add-move board move-as-keyword player))
+  (println (io/create-board-view board-with-move)))
 
 (defn -main
   "Tic Tac Toe program."
   [& args]
   (println "This is a Tic Tac Toe program.\n")
-  (println "Example:")
-  (println (board-to-string)))
+
+  (println "Example board:")
+  (def empty-board { :X #{} :O #{} })
+  (println (io/create-board-view empty-board))
+
+  (play-round empty-board :X))
```

Comment:
There are no tests around main. That's unacceptable. Please rewrite this doing TDD. If you're struggling to do so, please ask us or someone else for help.

Also, your tests for core.clj is just around what seems like private functions. There's not much value for those tests.

Reply:
Rewrite complete. I ended up getting all the output (for `play-round`, then `play-game`), and just checked if the string contained certain keywords (`Tic Tac Toe`, for example).